twofactor_privacyidea/lib/Provider/TwoFactorPrivacyIDEAProvider.php

```diff
@@ -139,7 +145,24 @@ private function triggerChallenges($username) {
 			if($result->getStatusCode() == 200) {
 				$body = json_decode($result->getBody());
 				if ($body->result->status === true) {
-					return $body->detail->messages;
+                    $detail = $body->detail;
+                    $this->detail = $detail;
+                    if (property_exists($detail, "transaction_ids")) {
+                        // TODO: What should we do, if there was more than one transaction ID?
+                        $this->transactionId = $detail->transaction_ids[0];
+                    }
+                    if (property_exists($detail, "attributes")) {
+                        $attributes = $detail->attributes;
+                        $this->hideOTPField = $attributes->hideResponseInput;
```

Comment:
In case ``attributes`` is present, is it guaranteed to contain ``hideResponseInput``? If not, we should probably add some kind of guard here.